enclave-runner/src/usercalls/mod.rs

```diff
@@ -165,18 +165,30 @@ pub trait SyncStream: 'static + Send + Sync {
 }
 
 
-trait SyncListener: 'static + Send + Sync {
-    fn accept(&self) -> IoResult<(FileDesc, Box<dyn ToString>, Box<dyn ToString>)>;
+/// SyncListener lets an implementation implement a slightly modified form of `std::net::TcpListener::accept`.
+/// The enclave may optionally request the local or peer addresses
+/// be returned in `local_addr` or `peer_addr`, respectively.
+/// On success, if `local_addr` and/or `peer_addr` are not `None`, they will point to an empty `String`.
+/// On success, user-space can fill in the strings as appropriate.
+///
+/// The enclave must not make any security decisions based on the local address received.
```

Comment:
Most of this documentation should be on the function

Reply:
Fixed in current push.